components/modal/index.en-US.md

```diff
@@ -32,6 +32,9 @@ and so on.
 | maskClosable | Whether to close the modal dialog when the mask (area outside the modal) is clicked | boolean   | true       |
 | style | Style of floating layer, typically used at least for adjusting the position. | object   | - |
 | wrapClassName | The class name of the container of the modal dialog | string   | - |
+| bodyStyle | Body style for modal body element. Such as height, padding etc. | object | {} |
```

Comment:
how to use bodyStyle? there is no example?

I tried to set height using it, doesn't work for me.

height="80%" works, but there is no mentioning of it on the document?